package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "crate-admin",
   "description": "Crate Admin UI",
-  "version": "1.15.0-dev",
+  "version": "1.14.1-dev",
```

Comment:
Is 1.14 already for CrateDB specific or do we have to go with 1.15 ?

Reply:
It is already used in 3.3.